src/devices/Amg88xx/Amg88xx.cs

```diff
@@ -0,0 +1,475 @@
+            return Temperature.FromDegreesCelsius(reading * ThermistorTemperatureResolution);
+        }
+
+        #endregion
+
+        #region Status
+
+        /// <summary>
+        /// Gets the temperature overflow flag from the status register
+        /// </summary>
+        /// <returns>Temperature overflow flag</returns>
+        public bool HasTemperatureOverflow()
+        {
+            return GetBit(Register.STAT, (byte)StatusFlagBit.OVF_IRS);
+        }
+
+        /// <summary>
+        /// Clears the temperature overflow flag in the status register
+        /// </summary>
+        public void ClearTemperatureOverflow()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, 1 << (byte)StatusClearBit.OVFCLR);
+        }
+
+        /// <summary>
+        /// Gets the thermistor overflow flag from the status register.
+        /// Note: the bit is only menthioned in early versions of the reference specification.
+        /// It is not clear whether this is a specification error or a change in a newer
+        /// revision of the sensor.
+        /// </summary>
+        /// <returns>Thermistor overflow flag</returns>
+        public bool HasThermistorOverflow()
+        {
+            return GetBit(Register.STAT, (byte)StatusFlagBit.OVF_THS);
+        }
+
+        /// <summary>
+        /// Clears the thermistor overflow flag in the status register
+        /// </summary>
+        public void ClearThermistorOverflow()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, 1 << (byte)StatusClearBit.OVFTHCLR);
+        }
+
+        /// <summary>
+        /// Gets the interrupt flag from the status register
+        /// </summary>
+        /// <returns>Interrupt flag</returns>
+        public bool HasInterrupt()
+        {
+            return GetBit(Register.STAT, (byte)StatusFlagBit.INTF);
+        }
+
+        /// <summary>
+        /// Clears the interrupt flag in the status register
+        /// </summary>
+        public void ClearInterrupt()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, 1 << (byte)StatusClearBit.INTCLR);
+        }
+
+        /// <summary>
+        /// Clears all flags in the status register
+        /// </summary>
+        public void ClearAllFlags()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, (1 << (byte)StatusClearBit.OVFCLR) | (1 << (byte)StatusClearBit.OVFTHCLR) | (1 << (byte)StatusClearBit.INTCLR));
+        }
+
+        #endregion
+
+        #region Moving average
+
+        /// <summary>
+        /// Get the state of the moving average mode
+        /// </summary>
+        /// <returns>True, if moving average mode is on</returns>
+        public bool GetMovingAverageModeState()
```

Comment:
@Ellerbach: I don't think that a property is the right chioce for this, because the reading the current state doesn't work with all revisions of the sensor. The current datasheet doesn't say that the state can be read while older revisions can. At least for the device that I used for testing I can confirm that reading the state isn't possible.
From my point of view a propery should always return the value that has been set using the same property. 
I also don't think that it would be a good idea trackig the set state in the binding as this would imply that I would make an assumption about the default state. This may change in a future revision of the device or there might be revisions out there which actually allow reading the current state. All this could end up in inconcsistency. 
That's the reason why I chose a Get-/Set-API. Waat's your advice?

Reply:
OK, then fair enough to keep it like this with the Get and Set. And you should add a comment about fact it may not run on al revisions properly as well.

Reply:
Consider calling it `UseMovingAverage` or similar, `MovingAverageModeState` suggests it will return some kind of enum

Reply:
I changed this into a property, too, to keep consistency across all the properties related to sensor configuration. I added a comment regarding the quirky behaviour, though, as reading the property may result always ```false``` even if it has just been set to ```true```.